x-pack/plugins/cross_cluster_replication/public/__jest__/client_integration/auto_follow_pattern_list.test.js

```diff
@@ -61,6 +61,48 @@ describe('<AutoFollowPatternList />', () => {
     });
   });
 
+  describe('when there are multiple pages of auto-follow patterns', () => {
+    let component;
+    let table;
+    let actions;
+    let form;
+
+    const autoFollowPatterns = [
+      getAutoFollowPatternMock({ name: 'unique', followPattern: '{{leader_index}}' }),
+    ];
+
+    for (let i = 0; i < 29; i++) {
+      autoFollowPatterns.push(
+        getAutoFollowPatternMock({ name: `${i}`, followPattern: '{{leader_index}}' })
+      );
+    }
+
+    beforeEach(async () => {
+      httpRequestsMockHelpers.setLoadAutoFollowPatternsResponse({ patterns: autoFollowPatterns });
+
+      // Mount the component
+      ({ component, table, actions, form } = setup());
+
+      await nextTick(); // Make sure that the http request is fulfilled
+      component.update();
+    });
+
+    test('pagination works', () => {
+      actions.clickPaginationNextButton();
+      const { tableCellsValues } = table.getMetaData('autoFollowPatternListTable');
+
+      // Pagination defaults to 20 auto-follow patterns per page. We loaded 30 auto-follow patterns,
+      // so the second page should have 10.
+      expect(tableCellsValues.length).toBe(10);
+    });
+
+    test('search works', () => {
+      form.setInputValue(component.find('input[type="search"]'), 'unique');
+      const { tableCellsValues } = table.getMetaData('autoFollowPatternListTable');
+      expect(tableCellsValues.length).toBe(1);
+    });
```

Comment:
@sebelga There are two other search tests like this one in the other tests files, and they are all failing based on the `tableCellsValues.length` remaining unchanged at 20. It looks like the input is being found and the change event is simulated, but it's not bubbling back up to the list component (which is responsible for handling it and updating the array of filtered items). Any ideas on what could be causing this?

Reply:
Have your tried adding `component.update()` after `form.setInputValue()`?

Reply:
Is there a reason why you decided to use `component.find('input[type="search"]')` instead of using a data-test-subj?

Reply:
> Have your tried adding component.update() after form.setInputValue()?

Yup, no change.

> Is there a reason why you decided to use component.find('input[type="search"]') instead of using a data-test-subj?

Fixed! I didn't realize this was possible until you suggested it and I looked at the EUI search bar config.